DeviceAgent.xcworkspace/xcshareddata/xcschemes/UITest.xcscheme

```diff
@@ -38,6 +38,16 @@
                ReferencedContainer = "container:DeviceAgent.xcodeproj">
             </BuildableReference>
          </TestableReference>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "4166C9AB22E7009800C8BEBF"
+               BuildableName = "TestAppUITests.xctest"
+               BlueprintName = "TestAppUITests"
+               ReferencedContainer = "container:DeviceAgent.xcodeproj">
+            </BuildableReference>
+         </TestableReference>
```

Comment:
This looks like it does not belong in this change set.

Reply:
I cleaned up all references of "TestAppUITests". It is not mentioned anywhere in the PR anymore